lib/internal/modules/cjs/loader.js

```diff
@@ -85,6 +85,8 @@ const {
 
 const isWindows = process.platform === 'win32';
 
+const filenameCache = Object.create(null);
```

Comment:
It might be better to call this something more descriptive like `parentResolveCache`.

Reply:
I named it `relativeResolveCache`. Are you fine with that?